### PR TITLE
fix(node): remove use of non existing `FunctionPrototypeApply` primordial

### LINKED ISSUE #21978: Next.js primordial error `FunctionPrototypeApply` is not a function
To reproduce:

1. `mkdir repro && cd repro`
1. `deno run -A --unstable npm:degit 'timlrx/tailwind-nextjs-starter-blog'`
2. Convert `next.config.js` to ESM 
3. `npm i`
4. `deno run -A --unstable-byonm ./node_modules/.bin/next dev`
5. Visit `http://localhost:3000`

In the wall of errors, you'll find:
```
 ○ Compiling / ...
 ⨯ ./data/logo.svg
TypeError: FunctionPrototypeApply is not a function
```

### PR DESCRIPTION
**Pull request by littledivy**
Fixes #21978

### CODE REVIEW

File: ext/node/polyfills/_util/_util_callbackify.ts [hunk 1]
```diff
@@ -27,7 +27,6 @@ import { primordials } from "ext:core/mod.js";
 const {
   ArrayPrototypePop,
   Error,
-  FunctionPrototypeApply,
```
Comment (dsherret) [inline ext/node/polyfills/_util/_util_callbackify.ts:30]: Can we add this primordial instead? I'm not sure how it works.

File: ext/node/polyfills/_util/_util_callbackify.ts [hunk 2]
```diff
@@ -112,10 +111,12 @@ function callbackify<ResultT>(
       throw new NodeInvalidArgTypeError("last");
     }
     const cb = (...args: unknown[]) => {
-      FunctionPrototypeApply(maybeCb, this, args);
+      // deno-lint-ignore prefer-primordials
+      maybeCb.apply(this, args);
```
[the remark this answers is not in the document]
Comment (petamoriken) [reply ext/node/polyfills/_util/_util_callbackify.ts:115]: How about using `ReflectApply` instead?

File: ext/node/polyfills/_util/_util_callbackify.ts (hunk 1 above)
Comment (littledivy) [reply ext/node/polyfills/_util/_util_callbackify.ts:30]: Used `ReflectApply` instead

**Review by dsherret — APPROVED** [review]
LGTM

**Pull request merged**